Cabal/doc/developing-packages.markdown

```diff
@@ -1476,13 +1482,45 @@ for these fields.
     `install-includes` is typically used to name header files that
     contain prototypes for foreign imports used in Haskell code in this
     package, for which the C implementations are also provided with the
-    package.  Note that to include them when compiling the package
-    itself, they need to be listed in the `includes:` field as well.
+    package. For example, here is a `.cabal` file for a hypothetical
+    `bindings-clib` package that bundles the C source code for `clib`:
+
+    ~~~~~~~~~~~~~~~~
+    include-dirs:     cbits
+    c-sources:        clib.c
+    install-includes: clib.h
+    ~~~~~~~~~~~~~~~~
+
+    Now any package that depends (directly or transitively) on the
+    `bindings-clib` library can include `clib.h` without needing to list it
+    explicitly in an `include` field.
```

Comment:
Well, there isn't actually a way to list it explicitly in `includes` (because you have no idea where it would have been installed).